src/Microsoft.OData.Core/ODataBatchWriter.cs

```diff
@@ -390,30 +373,13 @@ void IODataBatchOperationListener.BatchOperationContentStreamRequested()
         /// A task representing any action that is running as part of the status change of the operation;
         /// null if no such action exists.
         /// </returns>
-        Task IODataBatchOperationListener.BatchOperationContentStreamRequestedAsync()
-        {
-            // Write any pending data and flush the batch writer to the async buffered stream
-            this.StartBatchOperationContent();
-
-            // Asynchronously flush the async buffered stream to the underlying message stream (if there's any);
-            // then dispose the batch writer (since we are now writing the operation content) and set the corresponding state.
-            return this.rawOutputContext.FlushBuffersAsync()
-                .FollowOnSuccessWith(task => this.DisposeBatchWriterAndSetContentStreamRequestedState());
-        }
+        public abstract Task BatchOperationContentStreamRequestedAsync();

```

Comment:
As above, should be able to remove this line.

Reply:
see above.

---
In reply to: [140938120](https://github.com/OData/odata.net/pull/960#discussion_r140938120) [](ancestors = 140938120)